test/test.js

```diff
@@ -149,3 +154,24 @@ describe('Tests for password hashing functionality', () => {
     expect(isPassword).to.equal(false);
   });
 });
+
+describe('unit testing /users route', () => {
+  describe('testing with a dummy json', () => {
+    const user = {};
+    before(() => {
+      nock('http://testwiththisurl.com')
+        .get('/users')
+        .reply(201, user);
+    });
+    it('should return the expected json response', async () => {
+      const response = await supertest(app)
+        .get('/users');
+      assert(response.statusCode, 201);
+      expect(user).to.be.an('object');
+      expect(undefined).to.be.an('undefined');
```

Comment:
Add a test for success and failure on registration. You could test for the actually returned values in the object, that way your tests are more concrete.